x/leverage/keeper/keeper.go

```diff
@@ -432,7 +433,7 @@ func (k Keeper) LiquidateBorrow(
 	}
 
 	// apply liquidation incentive
-	reward.Amount = reward.Amount.ToDec().Mul(sdk.OneDec().Add(liquidationIncentive)).TruncateInt()
+	reward.Amount = bpmath.Mul(reward.Amount, liquidationIncentive).Add(reward.Amount)
```

Comment:
This is the only use of `Mul` (or `Quo`) I could find in the current diff

Interestingly enough, the original line had to be removed in my liquidation PR, because even the small `sdk.Dec` early rounding was contributing to dust issues. Now liquidation incentive is multiplied to a Dec without rounding back to Int.

I think it is possible to avoid 4-precision `Mul` and `Quo` altogether